cfme/tests/services/test_rest_services.py

```diff
@@ -234,12 +233,15 @@ def test_query_service_attributes(self, services, soft_assert):
         """
         outcome = query_resource_attributes(services[0])
         for failure in outcome.failed:
-            if failure.name == 'configuration_script' and BZ(
-                    1540250, forced_streams=['5.9', 'upstream']).blocks:
+            if failure.name == "reconfigure_dialog"and BZ(
+                    1663972, forced_streams=["5.9", "5.10", "upstream"]).blocks:
                 continue
-            if failure.name == 'metric_rollups' and BZ(
-                    1540254, forced_streams=['5.9', 'upstream']).blocks:
+            if failure.name == "configuration_script" and BZ(
+                    1540250, forced_streams=["5.9", "5.10", "upstream"]).blocks:
                 continue
+            if failure.name == "metric_rollups":
+                continue
```

Comment:
There was BZ1540254 for this subcollection and it's verified. Why do we need this `if`?

Reply:
`metric_rollups` is a special attribute, that requires some parameters to get a successful response. I am making changes that will handle this case.